File: Trabalho Cinema/Cinema/Models/Sessao.cs
using System.ComponentModel.DataAnnotations;
namespace Cinema.Models;


public class Sessao
{
    public int Id { get; set; }
    public int FilmeId { get; set; }
    public DateTime Data { get; set; }  // Usando DateTime para Data
    public string Horario { get; set; }
    public string Sala { get; set; }
    public decimal PrecoIngresso { get; set; }
    public int IngressosDisponiveis { get; set; }
    public Filme Filme { get; set; }  // Relação com Filme
}

File: my-app/src/models/Sessao.ts
// models/Sessao.ts
import { Filme } from "./Filme";

export interface Sessao {
  Id: number;
  FilmeId: number;
  Data: Date; 
  Horario: string;
  Sala: string;
  PrecoIngresso: number;
  IngressosDisponiveis: number;
  Filme?: Filme;
}